security-info/security-pack.md

```diff
@@ -95,7 +95,7 @@ to the Qubes mailing lists:
     very soon, the possibility of having to disclose any of the Qubes
     signing keys to anybody might have pretty serious consequences for those
     who decided to entrust Qubes with anything serious. And we would like to
-    somewhat minimize these consequences with this canary thing.
+    minimize these consequences with this canary thing.
```

Comment:
Go look at the rendered copy of this page. This is in the context of an email from Joanna reproduced in its entirety. I feel pretty strongly that the right thing to do here is keep the original "somehow" as-is.